server/src/main/java/org/eclipse/lsp/cobol/core/preprocessor/delegates/transformer/CobolLinesTransformation.java

```diff
@@ -25,6 +25,9 @@
  */
 public interface CobolLinesTransformation {
 
+  int START_INDEX_AREA_A = 4;
+  int END_INDEX_CONTENT_AREA_A = 10;
+  int INDICATOR_AREA = 7;
```

Comment:
You may move them to the `ProcessingConstants` class

Reply:
done.